sensorlib/src/util/CarlaUtils.py

```diff
@@ -67,7 +67,15 @@ def get_actor_bounding_box_points(carla_actor):
         :param carla_actor: The carla.Actor to obtain data from.
         :return: List of numpy.array containing the bounding box points in the world frame.
         """
-        bounding_box = carla_actor.bounding_box
+        
+        try:
+            bounding_box = carla_actor.bounding_box
+        except AttributeError:
+            #print("There is no bounding_box attribute, must be not Vehicles or Pedestrian, returning...")
```

Comment:
This feels like a silent failure. We are catching a exception here and then returning null. Is this the desired behavior? Do we expect this exception to happened in a correctly deployed setup and do we not want to long anything when it does happen?

Reply:
We still want the exception actually this was just a workaround. I kept the exception just for now to give more detail to the user.